layers/gpuav/core/gpuav_record.cpp

```diff
@@ -939,7 +939,7 @@ void Validator::PreCallRecordCmdDispatchIndirect(VkCommandBuffer commandBuffer,
         return;
     }
     auto &sub_state = SubState(*cb_state);
-    InsertIndirectDispatchValidation(*this, record_obj.location, sub_state, buffer, offset);
+    valcmd::Dispatch(*this, record_obj.location, sub_state, buffer, offset);
```

Comment:
this name is misleading, it seems like you forgot to add this to `PreCallRecordCmdDispatch()` as well

```suggestion
    valcmd::DispatchIndirect(*this, record_obj.location, sub_state, buffer, offset);
```

Reply:
I yeah I just removed the Indirect for no reason